web/server/models.py

```diff
@@ -44,12 +44,15 @@ class EventName(str, enum.Enum):
 
 
 class Modules(str, enum.Enum):
-    EDITOR = "editor"
-    DOCS = "docs"
-    PLANS = "plans"
-    TESTS = "tests"
-    AUDITS = "audits"
-    ERRORS = "errors"
+    EDITOR = "editor"  # include file editor and file explorer
+    DOCS = "docs"  # include docs
+    PLANS = "plans"  # include ability to run/apply plans
+    PLAN_PROGRESS = "plan-progress"  # include ability to see plan progress
+    TESTS = "tests"  # include ability to run tests
+    AUDITS = "audits"  # include ability to run audits
+    ERRORS = "errors"  # include ability to see errors
+    ENVIRONMENTS = "environments"  # include ability to see environments
+    ENVIRONMENT_PROD = "environment-prod"  # include ability to promote to prod
```

Comment:
This seems less like a module and more like a permissions.

Reply:
I was experimenting with this idea, but you are right, it is different concept so will remove.
We can add `like a permissions` later in separate PR if we need to